src/Stock.js

```diff
@@ -43,8 +50,16 @@ class Stock extends React.Component {
         let stockTimeValuesInner = [];
         let stockPriceValuesInner = [];
         let stockVolumeValuesInner = [];
+        let companyNameInner = [];
+        let company_52WeekHighInner = [];
+        let company_52WeekLowInner = [];
+        let companyDescriptionInner = [];
+        let priceToSalesRatioInner = [];
+        let profitMarginInner = [];
+        let RevenuePerShareTTMInner = [];
         let API_Mode = ''
         let API_Link = ''
+        let company_OverviewAPI = `https://www.alphavantage.co/query?function=OVERVIEW&symbol=${this.state.stockTicker}&apikey=${API_KEY}`
```

Comment:
For consistency, I think having all our API variables have API_ at the front makes sense, just to easily tell what is accessing external data. Renaming this to API_CompanyOverview would be my vote.

Reply:
All API variables have now been named accordingly!